src/maintainer/knowledge_base.rst

```diff
@@ -1118,6 +1119,36 @@ Note that this requires ``conda>=4.8``. If you want to support older conda versi
 the requirement should be changed from ``run`` to ``run_constrained``. Note that
 ``conda<4.8`` will ignore the condition if it's a ``run_constrained`` on ``__osx``.
 
+Compile for Apple's Metal framework
+-----------------------------------
+
+It is possible to compile against Apple's Metal framework. 
+However, you will likely need to use newer SDKs (11 and newer) for functionality.
+If in doubt, you can search the `phracker/MacOSX-SDKs <https://github.com/phracker/MacOSX-SDKs.git>`__ 
+repo for the exact symbols and their availability.
+
+
+Changing the Azure vmImage
+--------------------------
+
+Currently, the conda-forge's default vmImage for macOS is 10.15.
+If you increase the ``MACOSX_DEPLOYMENT_TARGET`` beyond this, the 
+``__osx`` run condition will prevent you from running tests.
+To use a newer vmImage, you can do it by editing ``conda-forge.yml`` 
+in the root of the repo like below. Note that the release after 10.15 is 11, 
```

Comment:
```suggestion
in the root of the repo like below. 
Note that the release after 10.15 is 11, 
```
I think a note should be slightly more prominent (Haven't checked though if this actually renders a linebreak).

Reply:
I will try to rerender the whole thing to check, thanks for the valuable comments!